gradle/dependencies.gradle

```diff
@@ -19,8 +19,6 @@ ext {
     logback           : "1.2.3",
     bytebuddy         : "1.10.18", // Also explicitly specified in buildSrc
     scala             : "2.11.12",
-    kotlin            : "1.4.0",
-    coroutines        : "1.3.0",
```

Comment:
`coroutines` is the library we instrument so it's more consistent for the version to be in the project